requirements.txt

```diff
@@ -18,3 +18,4 @@ distributed<2021.9
 pytest>=3.6,<7.0
 pytest-runner
 pytest-cov
+fsspec
```

Comment:
what's this one?

Reply:
this is a leftover, right? (@kenhester )